Process.c

```diff
@@ -706,6 +707,11 @@ void Process_printLeftAlignedField(RichString* str, int attr, const char* conten
    RichString_appendChr(str, attr, ' ', width + 1 - columns);
 }
 
+static inline bool is_dynamicColumn( const ProcessList* pl, int key) {
+   const DynamicColumn* column = Hashtable_get(pl->dynamicColumns, key);
+   return !column? true : false;
```

Comment:
Doesn't this negate the actual check?

Better without the `?:` altogether …
```suggestion
   return !!column;
```

Reply:
Perhaps even more clear as "return column != NULL;" ... ?

Reply:
Fine with both `return !!column;` and `return column != NULL;`, though I prefer the first variant …